src/shadowbox/server/manager_metrics.spec.ts

```diff
@@ -27,4 +30,16 @@ describe('PrometheusManagerMetrics', () => {
     expect(bytesTransferredByUserId['access-key-2']).toEqual(10000);
     done();
   });
+
+  it('getTunnelTimeByLocation', async (done) => {
```

Comment:
Let's have an example that shows that we can have multiple entries for each  country, asn or as_org.